### PR TITLE
add apple music and spotify 'support' (actually a redirect)

### PR DESCRIPTION
**Pull request by flavjyk4**
apple music and spotify redirects to this image (http.cat/451):

### CODE REVIEW

**Review by KwiatekMiki — COMMENTED** [review]
lgtm

File: src/modules/processing/services/applemusic.js
Comment (synzr) [inline src/modules/processing/services/applemusic.js:1]: two same files... why you can't do something like "services/451-redirect.js"?

File: src/modules/processing/match.js [hunk 1]
```diff
@@ -206,6 +208,12 @@ export default async function(host, patternMatch, lang, obj) {
                     ...patternMatch
                 });
                 break;
+            case "spotify":
```
Comment (synzr) [inline src/modules/processing/match.js:211]:
```js
case "spotify":
case "apple":
  r = await applemusic({}); // or await spotify({})
                            // or { urls; ["https://http.cat/451"] }
  break;
```

**Review by synzr — COMMENTED** [review]
this could be much better

**Review by synzr — COMMENTED** [review]
lgtm

**Comment by yeeterlol** [comment]
lgtm

**Review by yeeterlol — COMMENTED** [review]
lgtm

**Review by KwiatekMiki — COMMENTED** [review]
lgtm

**Pull request closed without merging**